controllers/storagecluster/storagecluster_controller.go

```diff
@@ -155,6 +155,53 @@ func (r *StorageClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		},
 	)
 
+	enqueueFromStorageProfile := handler.EnqueueRequestsFromMapFunc(
+		func(ctx context.Context, obj client.Object) []reconcile.Request {
+			spobj, ok := obj.(*ocsv1.StorageProfile)
+			if !ok {
+				return []reconcile.Request{}
+			}
+
+			// todo change according to standards
+			if spobj.Status.Phase != util.PhaseReady {
```

Comment:
I see the TODO here, question is, shouldn't we reconcile when phase is not ready generally? correct me if I'm reading this statement wrongly.

Reply:
I've made phase updates where I saw fit. I didn't see a need to add `[]conditionsv1.Condition` field to `StorageProfileStatus`.